blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -176,3 +179,8 @@ def test_distinct_on(sql):
     FROM {tbl}) AS anon_1 ORDER BY anon_1."A" ASC
     """.format(tbl=sql.name))
     assert odo(computation, tuple) == (('a', 1), ('b', 2))
+
+
+def test_function_in_expr():
+    s = symbol('s', 'int32')
+    assert odo(compute(s + 1, sa.func.max(1, bind=resource(db_url))), int) == 2
```

Comment:
This is already the case with tables and selects. I think having the bind as an argument to odo is a good idea; however, if the bind is part of the data, it should be used.